doc/examples/transform/plot_rescale.py

```diff
@@ -0,0 +1,31 @@
+"""
+=======
+Rescale
+=======
+
+The Rescale operation performs interpolation to 
```

Comment:
Wrap to 80.

I think we need to be careful about the language here.  Rescaling is seen in some communities as "rescaling of intensity".  So, perhaps something like:

"The rescale operation resizes an image by a given scaling factor.  The scaling factor can either be a single floating point value, or multiple values---one along each axis."

I think `interpolation=None` is now the default in matplotlib.

And yay for the first docathon commit!